Tally.cpp

```diff
@@ -0,0 +1,54 @@
+/* 
+ @file      Tally.cpp
+ @brief     contains functions for the Tally class
+ @author    Luke Eure
+ @date      January 8 2016
+*/
+
+#include "Tally.h"
+
+/*
+ @brief     constructor for Tally class
+*/
+Tally::Tally() {}
+
+/*
+ @brief     deconstructor
+*/
+Tally::~Tally() {}
+
+/*
+ @brief     add an amount to the tally and add its square to _tally_squared
+ @param     tally_addition an amount to be added
+*/
+void Tally::add(double tally_addition) {
```

Comment:
Could do this by overloading `operator+` to make it more natural.

Also: typically these types of operations should return `*this` so that they can be chained.